packages/admin/admin/src/index.ts

```diff
@@ -73,6 +73,7 @@ export { AsyncOptionsProps, useAsyncOptionsProps } from "./hooks/useAsyncOptions
 export { useStoredState } from "./hooks/useStoredState";
 export { InputWithPopper, InputWithPopperComponents, InputWithPopperComponentsProps, InputWithPopperProps } from "./inputWithPopper/InputWithPopper";
 export { InputWithPopperClassKey } from "./inputWithPopper/InputWithPopper.styles";
+export * from "./messages";
```

Comment:
what do you think about exporting a messages object so it can be used as `messages.edit`?

that would avoid having so many new exports

Reply:
I agree.. I've changed it. With IntelliSense it's now really nice to use.